clicache/integration-test2/Cluster.cs

```diff
@@ -71,7 +73,7 @@ private bool StartLocators()
             for (var i = 0; i < locatorCount_; i++)
             {
                 var locator = new Locator(this, new List<Locator>(),
```

Comment:
The second parameter here should be your `locators_` list so that each locators is aware of the previously created.

Reply:
Good catch!